### PR TITLE
doc-review identify pack name as known words

### PR DESCRIPTION
**Pull request by MosheEichler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/45735

## Description
Added the pack name to the known words in doc-review command

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

File: demisto_sdk/commands/doc_reviewer/doc_reviewer.py [hunk 1]
```diff
@@ -100,10 +100,10 @@ def find_known_words_from_pack(file_path: str) -> Tuple[str, list]:
                     return (packs_ignore_path, packs_known_words)
```
Comment (dorschw) [inline demisto_sdk/commands/doc_reviewer/doc_reviewer.py:100]: perhaps it's worth adding the pack name here too?

**Review by yaakovi — APPROVED** [review]
add to CHANGELOG

File: demisto_sdk/commands/doc_reviewer/tests/doc_reviewer_test.py [hunk 2]
```diff
@@ -723,9 +723,9 @@ def test_having_two_file_paths_not_same_pack(repo, mocker, first_file_content, s
 
 
 @pytest.mark.parametrize('known_words_content, expected_known_words',
-                         [(['[known_words]', 'wordament'], ['wordament']),
-                          (['[known_words]'], []),
-                          ([], [])])
+                         [(['[known_words]', 'wordament'], ['test_pack', 'wordament']),
+                          (['[known_words]'], ['test_pack']),
+                          ([], ['test_pack'])])
 def test_find_known_words_from_pack(repo, known_words_content, expected_known_words):
```
Comment (DinaMeylakh) [inline demisto_sdk/commands/doc_reviewer/tests/doc_reviewer_test.py:729]: Lets add some kind of documentation for this feature in the doc of this test so the expectation doesn't look suspicious

**Review by DinaMeylakh — COMMENTED** [review]
Looking good! 
Did you run locally to see it in action?

**Pull request merged**